src/schema/v2/partner.ts

```diff
@@ -35,13 +40,60 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
   },
 }
 
+const ArtistPartnerConnection = connectionWithCursorInfo({
```

Comment:
To avoid a circular dependency issue, this should move into the `fields` thunk several lines down.